pkg/core/blockchain.go

```diff
@@ -1918,6 +1918,9 @@ func (bc *Blockchain) GetPolicer() blockchainer.Policer {
 
 // GetBaseExecFee return execution price for `NOP`.
 func (bc *Blockchain) GetBaseExecFee() int64 {
+	if bc.BlockHeight() == 0 {
```

Comment:
Can we move this logic into `(ic *Context) BaseExecFee()`? We have `ic.Block` there that can be checked.

Reply:
It's not a one-line change, so fixed in a separate commit (6a4e312eac8db705630fded7d262b3d0e70cdf47).